koku/koku/trino_database.py

```diff
@@ -137,16 +137,8 @@ def executescript(trino_conn, sqlscript, *, params=None, preprocessor=None):
             p_stmt = p_stmt.removesuffix(";")
             # This is typically for jinjasql templated sql
             if preprocessor and params:
-                try:
-                    stmt, s_params = preprocessor(p_stmt, params)
-                except Exception as exc:
-                    LOG.warning(
-                        f"Preprocessor Error ({exc.__class__.__name__}) : {exc}{os.linesep}"
-                        f"Statement template : {p_stmt}{os.linesep}"
-                        f"Parameters : {params}"
-                    )
-                    exc_type = exc.__class__.__name__
-                    raise PreprocessStatementError(f"{exc_type} :: {exc}") from exc
```

Comment:
This try/except is here because jinjasql would fail to convert an iterable into the correct SQL format. jinjasql2 solved this issue, so the try/except was removed.